src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,139 @@
+package duke.ui;
+
+import duke.data.messages.Messages;
+
+/**
+ * Class that deals with interactions with the user of Duke.
+ *
+ * @author Wang Hong Yong
+ */
+public class Ui {
+
+    private static final String DIVIDER = "____________________________________________________________";
+    private static final String TAB = "  ";
+    private static final String LS = System.lineSeparator();
```

Comment:
You might want to give LS a more defined name and fully spell it out in caps, since it has a large scope and is a constant as well.